composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "friendsofphp/php-cs-fixer",
     "description": "A tool to automatically fix PHP code style",
     "license": "MIT",
+    "keywords": ["dev", "code style", "code-style", "fixer"],
```

Comment:
cs, sca, ...

also, are `code style` and `code-style` really treated differently?

Reply:
we coud add a lot of stuff here. the only important one is `dev`, to make composer work like described above.

we could add anything here. I am open to your suggestions :)

> also, are `code style` and `code-style` really treated differently?

I don't know

Reply:
I'd go with `fixer`, `standards` and `static analysis` only, the `dev` is not really valuable keyword.

Reply:
Composer expecta one of the following to trigger the desired behavior 


> dev, testing, static analysis

Reply:
adjusted per feedback